## Default parameters for autoware_error_monitor node
---
/**:
  ros__parameters:
    required_modules:
      manual_driving: []
      autonomous_driving: ['/autoware/control/autonomous_driving',
        '/autoware/control/command_gate',
        '/autoware/localization',
        '/autoware/map',
        '/autoware/perception',
        '/autoware/planning',
        '/autoware/sensing',
        '/autoware/system',
        '/autoware/vehicle'
      ]
      remote_control: ['/autoware/control/command_gate',
        '/autoware/control/remote_control',
        '/autoware/system',
        '/autoware/vehicle'
      ]
      safe_stop: ['/autoware/control/command_gate',
        '/autoware/map',
        '/autoware/system',
        '/autoware/vehicle'
      ]
      emergency_stop: ['/autoware/control/command_gate',
        '/autoware/system',
        '/autoware/vehicle',
      ]
